src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerRequestForInformation.java

```diff
@@ -160,24 +182,49 @@ private boolean isApplicantEmailValid(Applicant applicant) {
     }
 
     private List<String> isEmailValid(CaseData caseData, Applicant applicant) {
-        return isApplicantEmailValid(applicant) ? new ArrayList<>() : getValidationError(caseData, applicant);
+        return isApplicantEmailValid(applicant) ? new ArrayList<>() : getInvalidEmailError(caseData, applicant);
     }
 
     private List<String> areBothEmailsValid(CaseData caseData) {
```

Comment:
If you had a validateBothEmails method instead it would look bit simpler, like:
private void validateBothEmails(CaseData caseData, List<String> errors) {
    addEmailErrorIfInvalid(caseData.getApplicant1(), caseData, errors);
    addEmailErrorIfInvalid(caseData.getApplicant2(), caseData, errors);
}

private void addEmailErrorIfInvalid(Applicant applicant, CaseData caseData, List<String> errors) {
    if (!isApplicantEmailValid(applicant)) {
        errors.add(getInvalidEmailErrorString(caseData, applicant));
    }
}

Reply:
Good idea - I have refactored with this in mind.